src/Functions/generateSnowflakeID.cpp

```diff
@@ -165,18 +165,29 @@ class FunctionGenerateSnowflakeID : public IFunction
     {
         FunctionArgumentDescriptors mandatory_args;
         FunctionArgumentDescriptors optional_args{
-            {"expr", nullptr, nullptr, "Arbitrary expression"}
+            {"machine_id", nullptr, nullptr, "Optional machine ID"}
```

Comment:
Please retain the `expr` argument, we will need two arguments: `machine_id` and `expr`.

Reply:
Fixed, please review again, thank you!